keanu-project/src/test/java/io/improbable/keanu/util/json/JsonTest.java

```diff
@@ -0,0 +1,74 @@
+package io.improbable.keanu.util.json;
+
+import com.google.protobuf.util.JsonFormat;
+import io.improbable.keanu.KeanuSavedBayesNet;
+import io.improbable.keanu.network.BayesianNetwork;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.dbl.probabilistic.GaussianVertex;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.closeTo;
+import static org.hamcrest.collection.IsIn.isIn;
+import static org.hamcrest.core.Every.everyItem;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+public class JsonTest {
+
+    private static BayesianNetwork net;
+    private static Map<String, String> someMetadata;
+    private static ByteArrayOutputStream output;
+
+    @BeforeClass
+    public static void setup() throws IOException {
+        DoubleVertex gaussianVertex = new GaussianVertex(0.0, 1.0);
+        gaussianVertex.observe(0.5);
+        net = new BayesianNetwork(gaussianVertex.getConnectedGraph());
+
+        output = new ByteArrayOutputStream();
```

Comment:
Can we just put this on the declaration? 
`private static ByteArrayOutputStream output = new ByteArrayOutputStream();`